File: Python/interviewed-questions/same_elements.py
firstArray = ["alice", "bob", "trudy", "jack"]
secondArray = ["janet", "alice", "james", "jack"]
result = []

for i in range(len(firstArray)): # Iterate through the first array
    for j in range(len(secondArray)): # Iterate through the second array
        if firstArray[i] == secondArray[j]: # Name comparison between the two arrays
            result.append(firstArray[i])

print(result)

File: Python/interviewed-questions/same_elements.c
#include <stdio.h>

main()
{
	char firstArray[] = {"alice", "bob", "trudy", "jack"};
	char secondArray[] = {"janet", "alice", "james", "jack"};
	char result[] = {}
	
	for(i=0;i<firstArray.length;i++)
	{
		for(j=0;j<secondArray.length;j++)
		{
			if(firstArray[i]==secondArray[j])
			{
				result.push(firstArray[i]);
			}
		}
	}
	return result;
}
